main-page.js

```diff
@@ -30,19 +30,43 @@ class MainPage {
         noDataMessage.innerText = "Looks like you haven't had a ride yet. Let's start! To record a session click the record button on the bottom right corner";
     }
 
-    #onRecClick() {
-        this.#router.navigate("recording");
+    #onClickModalClose() {
+        document.getElementById("record-button").classList.remove("d-none");
```

Comment:
where do you create this record button? I don't think that you need to get it from the DOM by using getElementById. You should be able to use your record button variable here.